depthai_demo.py

```diff
@@ -216,6 +237,14 @@ def startLoop(self):
                             cv2.putText(frame, packet.stream_name, (25, 25), cv2.FONT_HERSHEY_SIMPLEX, 1.0, (0, 0, 255))
                             cv2.putText(frame, "fps: " + str(frame_count_prev[window_name]), (25, 50), cv2.FONT_HERSHEY_SIMPLEX, 1.0, (0, 0, 255))
                         else: # uint16
+                            if args['pointcloud'] and "depth_raw" in stream_names and "rectified_right" in stream_names and right_rectified is not None:
+                                if pcl_not_set:
+                                    pcl_converter = PointCloudVisualizer(depthai.get_right_intrinsic(), 1280, 720)
```

Comment:
I think we may be missing an import for this?

Reply:
Yes. I added that already